sqlite_test.go

```diff
@@ -514,7 +514,11 @@ func TestTrace(t *testing.T) {
 			t.Errorf("trace: err=%v, want %v", ev.err, errSubstr)
 		}
 		if ev.duration <= 0 || ev.duration > 10*time.Minute {
-			t.Errorf("trace: improbable duration: %v", ev.duration)
+			// The macOS clock appears to low resolution and so
+			// it's common to get a duration of exactly 0s.
+			if runtime.GOOS == "darwin" && ev.duration != 0 {
```

Comment:
shouldn't the first part be GOOS != darwin ?

Otherwise this error t.Errorf goes away on Linux.

Reply:
You marked this as resolved but I still don't understand it.

It looks wrong still.

Reply:
I forgot to git add the file: https://github.com/tailscale/sqlite/pull/30